scripts/run-sonarqube.sh

```diff
@@ -39,28 +39,12 @@ docker rm -f ${CONTAINER_NAME} || true
 docker run -d --net kind --name ${CONTAINER_NAME} -e SONAR_ES_BOOTSTRAP_CHECKS_DISABLE=true -p "${HOST_PORT}:9000" sonarqube:${SONAR_IMAGE_TAG}
 
 SONARQUBE_URL="http://localhost:${HOST_PORT}"
-echo "Waiting up to 4 minutes for SonarQube to start ..."
-n=0
-health="RED"
 set +e
-until [ $n -ge 24 ]; do
-    health=$(curl -s ${INSECURE} --user "${SONAR_USERNAME}:${SONAR_PASSWORD}" \
-        "${SONARQUBE_URL}/api/system/health" | jq -r .health)
-    if [ "${health}" == "GREEN" ]; then
-        echo " success"
-        break
-    else
-        echo -n "."
-        sleep 10s
-        n=$((n+1))
-    fi
-done
-set -e
-if [ "${health}" != "GREEN" ]; then
-    echo "SonarQube did not start, got health=${health}."
+if ! "${SCRIPT_DIR}/waitfor-sonarqube.sh" ; then
     docker logs ${CONTAINER_NAME}
     exit 1
-fi
+fi 
+set -e
```

Comment:
Is this actually needed any longer? I think you can remove line 42 now, no?

Reply:
Yes, good catch will provide an update.